<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBoard</class>
 <widget class="QMainWindow" name="MainBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>957</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
    border-image: url(:/images/images/Background_Main.jpg);
}

QTableWidget{
    background-color: rgb(0, 0, 0,50%);
	color: rgb(190, 157, 56);
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}

QHeaderView{
    background-color: rgb(0, 0, 0,50%);
	color: rgb(190, 157, 56);
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton{
border-image: url(:/images/images/PushButton.png) 0 0 0 0 stretch stretch;
font: 75 italic 14pt &quot;Cambria&quot;;
color: rgb(72, 52, 24);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>
    <item row="2" column="1" rowspan="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QTableWidget" name="tableWidget_players">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>asd</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>asd</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>asd</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>as</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>das</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>sada</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3" rowspan="7">
     <widget class="QWidget" name="widget_lowerRight" native="true">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_greatestTitles">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>The Greatest Knowledge</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>The Greatest Loot</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>The Greatest Leader</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>The Greatest Conqueror</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>The Greatest Adventurer</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>The Greatest City Conqueror</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>The Greatest Beating</string>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>Volkare Biddysen</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_cities">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Color</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Level</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Monsters</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>City Owner</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Color</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>Level</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>Monster</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>City Owner</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>BiddyCity</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>5 (10)</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>DuGaulingTown</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>6 (8)</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="7">
     <widget class="QWidget" name="widget_lowerLeft" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCustomPlot" name="customPlot" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>400</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0,75%);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_pushButtons" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_enterUserAction">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>User Action</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="pushButton_endGame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>230</horstretch>
              <verstretch>70</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>End Game</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_playPauseGame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Play/Pause Game
 Game Time: 00:00:00</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton_cityDiscovered">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>City Discovered</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
